### DISCUSSION TITLE
verified status but my organization does not get the benefits

### DISCUSSION THREAD

**Discussion by manathan1984** [community/community#57451] [GitHub Education]
### Hi everyone,

I am a university faculty with verified status, however, my organization does not get the benefits.

I have been using the benefits for a long time and for the first time, while my status is verified my organization cannot use the benefits (and I cannot submit a new application, because my last one was very recent). 

More specifically, in the application page [https://education.github.com/discount_requests/application](https://education.github.com/discount_requests/application), I received the following message:

> Congrats! Your request for [@BU-DiSC](https://github.com/BU-DiSC) was approved and your benefits are now available.

However, when I navigate to the setting of my organization [https://github.com/organizations/BU-DiSC/settings/billing](https://github.com/organizations/BU-DiSC/settings/billing) I see that I am on GitHub Free and on GitHub Teams as I used to be every time I got approved. 

I already waited two weeks in my ticket in GitHub support and they told to ask in the GitHub community. 

Looking forward to getting some support on my issue,
Manos

**Comment by manathan1984** [comment]
Similar threads to date: #57833 #49950 #57803

**Comment by bobturneruk** [comment]
The excellent @robadob has found this https://education.github.com/globalcampus/teacher - I'm checking if it works on a test organisation.
